src/psyclone/psyir/frontend/fparser2.py

```diff
@@ -1164,11 +1164,15 @@ def process_declarations(self, parent, nodes, arg_list):
 
                 if initialisation:
                     if has_constant_value:
-                        # If it is a parameter parse its initialization
-                        tmp = Assignment()
+                        # If it is a parameter parse its initialization into
```

Comment:
Could you please add a blank line before the final ''' in the docstring.

Reply:
Done